componentLibrary.css

```diff
@@ -0,0 +1,57 @@
+:root {
+    --width-sm: 3rem;
+    --height-sm: 3rem;
+    --width-md: 5rem;
+    --height-md: 5rem;
+    --width-lg: 8rem;
+    --height-lg: 8rem;
+    --width-xl: 10rem;
+    --height-xl: 10rem;
+}
+
+.heading {
+    text-align: center;
+}
+
+/* *****avatar***** */
+.avatar-container {
+    max-width: 100%;
+    border: 1px solid black;
+    margin: 2rem;
+    padding: 2rem;
+    display: flex;
+    flex-direction: row;
```

Comment:
use flex property gap for adding space between two elements and use shorthand property flex-flow for flex-direction and flex-wrap like flex-flow: row wrap

Reply:
ok, i will do that